app/utils/geostore.py

```diff
@@ -34,7 +40,23 @@ async def _get_gfw_geostore(geostore_id: UUID) -> GeostoreCommon:
     return geostore_common
 
 
-async def get_geostore(geostore_id: UUID, geostore_origin: str) -> GeostoreCommon:
+def check_all_geostores():
```

Comment:
Similar to my comment above, do we want to be making a separate function like this for every feature flag, or just one function where we pass the feature flag name?